### DISCUSSION TITLE
"dotnet sergen g" not working on existing project from online

### DISCUSSION THREAD

**Discussion by AaronRiboy** [serenity-is/Serenity#6607] [Help] [answered]
<img width="854" alt="image" src="https://user-images.githubusercontent.com/29535650/205851350-12944d3a-53c1-45ed-bfdb-a23c9418e7b0.png">

Bought a SmartERP template from online.Its a serenity framework. For some reason its not allowing me to add new table. How to resolve this?? Does anyone know this?Thank you

**Comment by VictorTomaili** [comment]
This is related to your computer configuration. We did not provide support for SmartERP or some project which is use serenity platform as base. This error is not related to the sergen itself. Please check read the error or ask your IT.

**Reply by AaronRiboy** [reply]
Noted with thanks

**Comment by reach2rv** [comment, marked as answer]
@AaronRiboy 

Right click on zip folder you have downloaded and open properties there you will find options to unblock. Check on that and click ok. Post this extract files so you will not face this issue. This usually happens with downloaded files on windows.

**Reply by AaronRiboy** [reply]
Thank you..its solved
